src/api/bank-accounts/bank-account-requests.md

```diff
@@ -0,0 +1,112 @@
+---
+layout:  default
+grand_parent: API Reference
+parent: Bank Accounts
+title: Bank Account Requests
+permalink: /api/bank-account-requests
+---
+
+# Bank Account Requests
+{:.no_toc}
+
+Bank Account Requests facilitate user authorization of access to bank accounts.
+Typically the authorization will follow an OAuth flow where the user is
+redirected to a third-party system to authenticate and select one or more bank
+accounts. After a bank account request has been completed, one or more
+Centrapay Bank Account resources may be created and long-lived access tokens
+may be stored against those Bank Accounts.
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Models
+
+### Bank Account Request
+{% h4 Mandatory Fields %}
+
+|          Field           |        Type        |                                             Description                                              |
+| :----------------------- | :----------------- | :--------------------------------------------------------------------------------------------------- |
+| id                       | String             | The Bank Account Request's unique identifier.                                                        |
+| accountId                | String             | The id of the owning Centrapay [Account][].                                                          |
+| type                     | String             | The type of Bank Account Request. See [Bank Account Request Types][].                                |
+| status                   | String             | The current status of the Bank Account Request. Supported values are `created`, `failed` and `done`. |
+| createdAt                | {% dt Timestamp %} | When the Bank Account Request was created.                                                           |
+| createdBy                | {% dt CRN %}       | The User or API Key that created the Bank Account Request.                                           |
+| updatedAt                | {% dt Timestamp %} | When the Bank Account Request was updated.                                                           |
+| updatedBy                | {% dt CRN %}       | The User or API Key that updated the Bank Account Request.                                           |
+| authorizationRedirectUrl | String             | A redirect url to begin the OAuth 2 flow.                                                            |
```

Comment:
Perhaps this description could just say: 'A redirect URL to begin the authorization flow.'

Reply:
Have change it to something similar.

Reply:
@IsaacMurtagh it looks like it says the same thing?